civitai-downloader-v2/tests/api/test_models.py

```
@@ -0,0 +1,206 @@
+"""
+Test model-related API endpoints.
+"""
+import pytest
+from fastapi.testclient import TestClient
+from tests.conftest import TestConstants, validate_model_structure
```

Ruff (0.12.2)

`tests.conftest.TestConstants` imported but unused

Remove unused import: `tests.conftest.TestConstants`

(F401)